jsonforms-property-view/src/browser/widget.tsx

```diff
@@ -10,13 +10,7 @@
  ********************************************************************************/
 import { JsonFormsCore, JsonSchema, UISchemaElement } from '@jsonforms/core';
 import { JsonForms } from '@jsonforms/react';
-import {
-    JsonFormsStyleContext,
-    StyleContext,
-    vanillaCells,
-    vanillaRenderers,
-    vanillaStyles
-} from '@jsonforms/vanilla-renderers';
+import { JsonFormsStyleContext, StyleContext, vanillaCells, vanillaRenderers, vanillaStyles } from '@jsonforms/vanilla-renderers';
```

Comment:
this looks broken, shouldn't this be formatted as a block?

Reply:
The prettier config allows `printWidth: 140`, and this import line has a length of `131`, so I'd say it is okay this way.
I doublechecked with other projects, e.g. the GLSP config allows the same `printWidth` as well. The default prettier config would only allow a width of `80`.